src/ql/ir/old_to_new.cc

```diff
@@ -921,7 +921,8 @@ Ref convert_old_to_new(const compat::PlatformRef &old) {
 
     // Check the result.
     QL_DOUT("Result of old->new IR platform conversion:");
-    QL_IF_LOG_DEBUG(ir->dump_seq());
+    QL_DOUT("(disabled)");
```

Comment:
use #if/#else. I guess what we really need is finer control over log output per 'module' like Python has

Reply:
Yes, agreed.
Don't know solution yet for this.
Updated the idiom to simulate enable/disable per file.
This allows finer control over endless dumping, at least
because the debug output got clogged up by those dumps.